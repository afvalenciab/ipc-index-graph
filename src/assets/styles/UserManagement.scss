.container--management {
  display: grid;
  grid-template: 80px minmax(auto, 1fr) 30px / 100vw;
  grid-gap: 10px;
  min-height: 100vh;
  box-sizing: border-box;
}