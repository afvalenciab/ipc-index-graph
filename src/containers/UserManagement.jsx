import React from 'react';
import Header from '../components/Header';
import UserControls from '../components/UserControls';
import Footer from '../components/Footer';
import '../assets/styles/UserManagement.scss';

const UserManagement = () => {
  return (
    <main className='container--management'>
      <Header />
      <UserControls />
      <Footer />
    </main>
  );
};

export default UserManagement;
