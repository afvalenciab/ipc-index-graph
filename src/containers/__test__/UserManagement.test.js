import React from 'react';
import { create } from 'react-test-renderer';
import ProviderMock from '../../__mocks__/ProviderMocks';
import UserManagement from '../UserManagement';

describe('UserManagement testing', () => {
  test('Match Snapshot', () => {
    const userManagement = create(
      <ProviderMock>
        <UserManagement />
      </ProviderMock>
    );
    expect(userManagement.toJSON()).toMatchSnapshot();
  });
});
