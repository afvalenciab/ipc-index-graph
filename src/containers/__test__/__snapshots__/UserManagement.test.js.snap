// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserManagement testing Match Snapshot 1`] = `
<main
  className="container--management"
>
  <header
    className="header"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      <h1>
        IPC
        <span>
          Index
        </span>
      </h1>
    </a>
    <nav
      className="menu"
    >
      <ul>
        <li>
          <a
            href="/login"
            onClick={[Function]}
          >
            Ingresar
          </a>
        </li>
        <li
          className="menu--register"
        >
          <a
            href="/register"
            onClick={[Function]}
          >
            Crear cuenta
          </a>
        </li>
      </ul>
    </nav>
  </header>
  <section
    className="user--controls"
  >
    <h2>
      Administración de Usuarios
    </h2>
    <div
      className="user--controls_table"
    >
      <div
        className="user--controls_header"
      >
        <p>
          Correo electrónico
        </p>
        <p>
          Ingresos fallidos
        </p>
        <p>
          Admon
        </p>
        <p>
          Eliminar
        </p>
      </div>
      <div
        className="user--controls_list"
      />
    </div>
  </section>
  <footer
    className="footer"
  >
    <p>
      IPC Index - Indice de Precios y Cotizaciones.
    </p>
    <p>
      Hecho por Andres Valencia
    </p>
  </footer>
</main>
`;
